Documentation/BuildInstructionsLadybird.md

```diff
@@ -143,8 +144,14 @@ WSL1 is known to have issues. If you run into problems, please use WSL2.
 
 MinGW/MSYS2 are not supported.
 
-Native Windows builds are not supported.
+# Native (Visual Studio Only)
+ you need choclatey package manager then run the commands all in an admin terminal of powershell
+install pkgConfig
+```
+choco install pkgconfiglite -y
```

Comment:
Why is chocolatey needed for pkgconfig? I thought vcpkg shipped their own version. If really needed we could probably piggy-back off their binary

Reply:
I tried to get cmake to see the one with vcpkg but it wouldn't work so found a way to install it onto the system and I found that way

Reply:
Fair enough. we can leave it as a suggestion. I don't know enough about windows dev environments either way.

Reply:
> Why is chocolatey needed for pkgconfig? I thought vcpkg shipped their own version. If really needed we could probably piggy-back off their binary

vcpkg version (pkgconf) does not work for me on Mac, it does not output "private" cflags/libs for some reason and I did install real pkg-config (with glib2.0 dependency) to correctly link libraries. I'm pretty sure the same issue happens on Windows.